src/items/rqgItem.ts

```diff
@@ -96,6 +100,101 @@ export class RqgItem extends Item {
     });
   }
 
+  protected _onCreate(
+    data: ItemDataSource,
+    options: DocumentModificationOptions,
+    userId: string
+  ): void;
```

Comment:
This is just duplicated code I think?

Reply:
VS code autogenerated it this way with both signatures.  Removing the one you pointed out here still works.  I guess I didn't know why it was doing that.